packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="cpprestsdk.v140.windesktop.msvcstl.dyn.rt-dyn" version="2.9.1" targetFramework="native" />
-  <package id="Microsoft.Azure.RemoteRendering.Cpp" version="0.1.52" targetFramework="native" />
+  <package id="Microsoft.Azure.RemoteRendering.Cpp" version="0.1.66" targetFramework="native" />
```

Comment:
We didn't release 0.1.66. Possibly the resolution will automatically pull 0.1.67, but it's probably best to explicitly say 0.1.67.

Reply:
That's interesting. Nuget still seems to be pulling a valid package with that config, which it also thinks to be 0.1.66 from what I can tell. I can also go to 0.1.65 to be safe, since the fresnel feature was in there as well.

Reply:
Moved to 0.1.65 for safe measure.

Reply:
Please update also cgmanifest.json

Reply:
Ah damn, missed that one. Done :)